lib/chromium/webconsole.js

```diff
@@ -283,6 +293,19 @@ var ChromiumConsoleActor = ActorClass({
       result = response.result;
     }
 
+    // Set $_ for future availability.
+    let lastResult = response.result.objectId;
+    // If not an object, provide an descriptive error to inform the user.
+    if (!lastResult) {
+      let boxed = yield this.rpc.request("Runtime.evaluate", {
```

Comment:
The other issue on canary I see is that this statement never shows up in the console frontend (even when it is requested/received from backend)